routes/web.php

```diff
@@ -24,9 +24,17 @@
 Route::delete('/period/{id}', 'PeriodController@delete')->name('period.delete');
 
 // Forum routes
-Route::get('/post/index', 'PostController@index')->name("post.index");
-Route::get('/post/new', 'PostController@new')->name("post.new");
-Route::get('/post/show/{id}', 'PostController@show')->name("post.show");
-Route::post('/post/save', 'PostController@save')->name("post.save");
-Route::delete('/post/destroy/{id}', 'PostController@destroy')->name("post.destroy");
+Route::get('/post/index', 'PostController@index')->name('post.index');
+Route::get('/post/new', 'PostController@new')->name('post.new');
+Route::get('/post/show/{id}', 'PostController@show')->name('post.show');
+Route::post('/post/save', 'PostController@save')->name('post.save');
+Route::delete('/post/delete/{id}', 'PostController@delete')->name('post.delete');
+Route::post('/post/edit/{id}', 'PostController@edit')->name('post.edit');
+Route::patch('/post/update/{id}', 'PostController@update')->name('post.update');
+
+// Comments routes
+Route::post('/comment/save/{post_id}', 'CommentController@save')->name('comment.save');
```

Comment:
Do not send post_id as a parameter. Send it via query instead. It means:

/comment/save alone is the route

and the controller should receive a request

save(Request req) {
  req['post_id'] // there's the post id
}

Reply:
Done